eras/babbage/test-suite/cardano-ledger-babbage-test.cabal

```
@@ -91,7 +91,7 @@ test-suite cardano-ledger-babbage-test
         cardano-ledger-binary:testlib,
         cardano-ledger-alonzo-test,
         cardano-ledger-babbage:{cardano-ledger-babbage, testlib},
-        cardano-ledger-babbage-test >=1.1,
+        cardano-ledger-babbage-test,
```

This is needed, since library does not depend on this package
```suggestion
        cardano-ledger-babbage-test >= 1.1,
```

---

How so? This library is itself

---

:man_facepalming: 

Did not look at the cabal file name when was making this suggestion, just looked into the dependencies of the library. Somehow it does not depend on itself, outrageous! :smile: